pkg/application/inject/fuse/container_injector.go

```diff
@@ -17,84 +17,55 @@ limitations under the License.
 package fuse
 
 import (
-	"fmt"
 	"github.com/fluid-cloudnative/fluid/pkg/common"
-	"github.com/fluid-cloudnative/fluid/pkg/scripts/poststart"
-	"github.com/fluid-cloudnative/fluid/pkg/utils"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
 )
 
```

Comment:
How about adding comments to the function name to improve the understanding of the logic?

Reply:
Done